scripts/travis/external_build.sh

```diff
@@ -52,29 +52,31 @@ if [ "${BUILD_TARGET}" = "" ]; then
     exit 1
 fi
 
-sudo apt-get install awscli
+BUILDID=${TRAVIS_JOB_NUMBER/./000}
```

Comment:
I'd like to see a comment example what is in TRAVIS_JOB_NUMBER and what is in BUILDID after replacement (that is not well-known feature)

Reply:
will do in subsequent PR.